src/index/script/HeadsUpDisplay/StatusWidget/_StatusWidget.coffee

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Globals from '../../../../_common/script/Globals.coffee'
+
+export default class StatusWidget extends React.Component
+
+	############################################################################
+
+	constructor: (props) ->
+		super props
+
+	############################################################################
+
+	render: =>
+		<form onSubmit={this.handleSubmit}>
+			<button type="submit" className="widget" id="status-widget">
+				{this.props.status}
+			</button>
+		</form>
+
+	#///////////////////////////////////////////////////////////////////////////
+	
+	handleSubmit: (e) =>
+		e?.preventDefault()
+		switch this.props.status
+			when Globals.STATUS_PLACETILE
```

Comment:
Why use a 'switch' when there is only one 'when'?  Are more coming?

Reply:
There will be lots more statuses coming, and this switch statement is the signal for where to put the high-level logic handling them.